core/src/main/java/uk/ac/bbsrc/tgac/miso/core/manager/MisoRequestManager.java

```diff
@@ -2565,4 +2593,24 @@ public void deleteBox(Box box) throws IOException {
       throw new IOException("No boxStore available. Check that it has been declared in the Spring config.");
     }
   }
+
+  @Override
+  public long saveSequencerServiceRecord(SequencerServiceRecord record) throws IOException {
+    return sequencerServiceRecordStore.save(record);
```

Comment:
MISO convention for RequestManager has an initial check for if xStore is null (add to this and three methods below)